lib/JsonpMainTemplate.runtime.js

```diff
@@ -3,49 +3,52 @@
 	Author Tobias Koppers @sokra
 */
 /*globals hotAddUpdateChunk parentHotUpdateCallback document XMLHttpRequest $require$ $hotChunkFilename$ $hotMainFilename$ */
+"use strict";
```

Comment:
Please don't refactor this file. It should be in low level js as it's embedded into the runtime code